sale_credit_point/models/sale_order.py

```diff
@@ -54,5 +54,5 @@ def action_cancel(self):
                 sale.partner_id.credit_point_increase(
                     sale.amount_total, _("Sale Order canceled"))
                 sale.partner_id.update_history(
-                    sale.amount_total, "increase", _("Sale Order canceled"))
```

Comment:
I don't get this change: the card says that we should NOT update history on cancel.

Reply:
if it is OK just not update history then I can fix pr.
if we still need some records for history then we need to add a new operation that not affects the yearly_point_increase calculation

Reply:
So what is the correct solution in your opinion?

Reply:
here? keep this change, sounds fine.

Reply:
Yes. In `def action_cancel(self)`
But IMHO store in the history information about cancelled SO is not history made for?